.github/workflows/send-email.yml

```diff
@@ -0,0 +1,95 @@
+name: Send communications
+on:
+  push:
+    branches:
+      - production
+    paths:
+      - 'apps/dialtone-documentation/docs/about/whats-new/posts/*.md'
+
+env:
+  HUSKY: 0
+
+jobs:
+  changed-files:
+    name: Get changed files
+    runs-on: ubuntu-latest
+    outputs:
+      added-posts: ${{ steps.changed-blog-posts.outputs.added_files }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Get changed blog posts
+        id: changed-blog-posts
+        uses: tj-actions/changed-files@v42
+        with:
+          files: 'apps/dialtone-documentation/docs/about/whats-new/posts/*.md'
+          json: true
+          escape_json: false
+      - name: List all changed files
+        run: echo '${{ steps.changed-blog-posts.outputs.all_changed_files }}'
+
+  send-email:
+    name: Send email
+    runs-on: ubuntu-latest
+    needs: [changed-files]
+    strategy:
+      matrix:
+        files: ${{ fromJSON(needs.changed-files.outputs.added-posts) }}
+    env:
+      FROM_MAIL: ${{ secrets.MAIL_ACCOUNT }}
+      TO_MAIL: dialtone-users@dialpad.com
+      FILE_PATH: ${{ matrix.files }}
+      REPLY_TO: dialtone@dialpad.com
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Get link from file name
+        id: get-post-url
+        run: |
+          PREFIX="https:\/\/dialtone.dialpad.com"
+          echo "POST_URL=$(echo $FILE_PATH | sed -e "s/apps\/dialtone-documentation\/docs/$PREFIX/" -e 's/md/html/')" >> $GITHUB_ENV
+
+      - name: Get heading from blog post
+        id: get-heading
+        run: |
+          echo "BLOG_POST_HEADING=$(sed -n 's/^heading: *//p' $FILE_PATH)" >> $GITHUB_ENV
+
+      # This will do the following:
+      # - Remove frontmatter.
+      # - Remove script setup.
+      # - Remove blog post component.
+      # - Remove comments.
+      # - Remove empty lines at the beginning of the file.
+      # - Ellipsis the article to the first 10 lines.
+      # - Append the URL of the article at the end of the article.
+      - name: Process Markdown file
+        run: >
+          sed -i
+          -e '/^---$/,/^---$/d'
+          -e '/^<script setup>$/,/^<\/script>$/d'
+          -e '/^<\/*BlogPost.*$/d'
+          -e '/^<!--.*$/d'
+          -e '/./,$!d'
+          $FILE_PATH &&
+          sed -i 10q $FILE_PATH &&
+          echo -e "... Read the full article: $POST_URL" >> $FILE_PATH;
+
+      - name: Send email
+        uses: dawidd6/action-send-mail@v3
+        with:
+          connection_url: ${{ secrets.MAIL_CONNECTION }}
+          subject: "Dialtone: New blog post '${{ env.BLOG_POST_HEADING }}'"
+          to: ${{ env.TO_MAIL }}
+          from: Dialtone <${{ env.FROM_MAIL }}>
+          html_body: file://${{ env.FILE_PATH }}
+          bcc: ${{ env.FROM_MAIL }}
+          reply_to: ${{ env.REPLY_TO }}
```

Comment:
Note, Some receivers may reject this cross-site behavior. If it works for us, great. If not, consider sending the email to dialtone@dialpad.com with each recipient on BCC. Or keeping it simple and sending from dialtone.updates@gmail.com to dialtone-users@dialapad.com.

Reply:
I don't understand completely what's the suggestion here.

We're currently sending from dialtone.updates@gmail.com to dialtone-users@dialpad.com.

The reply_to: dialtone@dialpad.com is so users can respond to an email all dialtone members can see, otherwise the reply will end up on an inbox that not many people can see right now.

You're suggesting to change that reply_to?